lessons/lesson-template.md

```diff
@@ -0,0 +1,74 @@
+- title: 
+- tags: 
+
+###Standards:
+(Overarching learning goals, e.g. 'Students become comfortable with Angular')
+
+### Objectives
+(Concrete, measurable tasks and learnings that map back to the overarching learning goals)
+Eg. By the end of the lesson, students will be able to:
+* Facilitate a daily standup
+* Define sprints and minimum viable products
+
+### Schedule
+* Warmup: link to challenge problem
+* Morning lecture: e.g. Intro to Arrays
+* Morning exercises: Link to exercises
+* Midday checking: link to checking
+* Lunch
+* Afternoon lecture: e.g. More Arrays
+* Afternoon exercises: Link to exercises
+* Review
+
+### Resources
```

Comment:
Maybe can reference them in line? Think it would be nice though to have them all in one place - the idea will be to also include extra links that may not necessarily cover in class